prefect_gcp/credentials.py

```diff
@@ -70,6 +70,13 @@ class GcpCredentials(Block):
         service_account_file: Path to the service account JSON keyfile.
         service_account_info: The contents of the keyfile as a JSON string / dictionary.
         project: Name of the project to use.
+
+    Example:
+        Load stored GCP credentials:
+        ```python
+        from prefect_gcp import GcpCredentials
+        gcp_credentials_block = GcpCredentials.load("MY_BLOCK_NAME")
```

Comment:
The UI is expecting `"BLOCK_NAME"` for interpolation.
```suggestion
        gcp_credentials_block = GcpCredentials.load("BLOCK_NAME")
```